migrations/20220614074223_add_ip_address_to_audit_log.postgres.up.sql

```diff
@@ -0,0 +1,3 @@
+-- Add IP Address to audit log
+ALTER TABLE auth.audit_log_entries
+ADD COLUMN IF NOT EXISTS ip_address VARCHAR(32) NOT NULL DEFAULT '';
```

Comment:
Initial considerations was to take the closest exponent of 2 which could store  IPv4 + CIDR prefix(e.g. 192.168.123.111/254)  as well as IPv6 addresses(2001:0db8:85a3:0000:0000:8a2e:0370:7334). After sleeping on it and doing a search I realised that I've neg that the IPv6 size can [potentially be up to 45 bytes](https://stackoverflow.com/questions/3455320/size-for-storing-ipv4-ipv6-addresses-as-a-string) and I'd also made an error since  32 bytes doesn't account for the colons

Perhaps we could go with 64 bytes?